rma_sale/views/sale_portal_template.xml

```diff
@@ -15,7 +15,7 @@
                     class="btn-close"
                     aria-label="Close"
                     data-bs-dismiss="modal"
-                >&amp;times;</button>
+                />
```

Comment:
Why removing the word `times`?

Reply:
This is to avoid showing the icon 2 times (I'm seeing that it happens also in v16, I'm going to fix it too).
![ejemplo](https://github.com/user-attachments/assets/8d8434d2-7e6c-4a5a-b591-403cbcd75fec)